direct/src/directbase/TestStart.py

```diff
@@ -1,8 +1,5 @@
 print('TestStart: Starting up test environment.')
 
-from panda3d.core import *
-
-from direct.showbase.PythonUtil import *
```

Comment:
PythonUtil puts things in builtins, that's why they are there in these legacy scripts - no need to break that usage

Reply:
I guess we can just turn this into `import direct.showbase.PythonUtil`